anti-analysis/anti-vm/vm-detection/check-for-sandbox-username-hostname.yml

```diff
@@ -37,33 +41,37 @@ rule:
           description: Gookit Username Check
         - string: /FORTINET/i
           description: Shifu Username Check
-        - string: /John\sDoe/i
+        - string: /John\s?Doe/i
           description: Emotet Username Check
         - string: /Emily/i
           description: Trickbot Downloader Username Check
-        - string: /HANSPETER\-PC/i
+        - string: /HANSPETER(-PC)?/i
           description: Trickbot Downloader Username Check
         - string: /HAPUBWS/i
           description: Trickbot Downloader Username Check
         - string: /Hong\sLee/i
           description: Trickbot Downloader Username Check
-        - string: /IT\-ADMIN/i
+        - string: /IT-ADMIN/i
           description: Trickbot Downloader Username Check
-        - string: /JOHN\-PC/i
+        - string: /JOHN(-PC)?/i
           description: Trickbot Downloader Username Check
         - string: /Johnson/i
           description: Trickbot Downloader Username Check
         - string: /Miller/i
           description: Trickbot Downloader Username Check
-        - string: /MUELLER\-PC/i
+        - string: /MUELLER(-PC)?/i
           description: Trickbot Downloader Username Check
         - string: /Peter\sWilson/i
           description: Trickbot Downloader Username Check
         - string: /SystemIT/i
           description: Trickbot Downloader Username Check
         - string: /Timmy/i
           description: Trickbot Downloader Username Check
-        - string: /WIN7\-TRAPS/i
+        - string: /WIN7-TRAPS/i
           description: Trickbot Downloader Username Check
         - string: /WDAGUtilityAccount/i
           description: Windows Defender Application Guard
+        - string: /HAL9TH/i
+          description: Windows Defender Emulator
+        - string: /REM/i
+          description: SANS REM VM
```

Comment:
strings with less than 4 characters won't be extracted, can you add a character or remove this entry?

Reply:
Removed.